src/models/thelio-mira-b4-n3/repairs.md

```diff
@@ -0,0 +1,613 @@
+    - `PMB1`/`MOTHERBOARD` - to the first four pins of the `F_PANEL` header at the bottom right of the motherboard, highlighted green below.
+        - On the Thelio Io board, the wire color order (from top to bottom) is red, black, blue, black.
+        - On the motherboard, the wire color order (from left to right) is red, black, blue, black.
+    - `USB` - to the `USB_3_4` header on the bottom edge of the motherboard, highlighted yellow below.
+        - A small clip needs to be held down to unplug this cable from the Thelio Io board.
+        - This port can alternatively be connected to the `USB_5_6` header if the `USB_3_4` header is connected to the wireless card.
+        
+    ![Motherboard power button & USB headers](./img/motherboard-power-usb.webp)
+
+## Replacing the SATA backplane:
+
+The SATA backplane provides two 2.5" SATA data and power connectors for the [2.5" drive cage](#addingremoving-25-storage-drives).
+
+**Part numbers:**
+- The part number and version for the SATA backplane is [thelio-io-sata_2.2](https://github.com/system76/thelio-io-hardware/tree/91814370002f762ac01cd1dd99dcbb43336352ab/pcb-thelio-io-sata).
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 30 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>  
+
+### Steps to replace the SATA backplane:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), then [remove the 2.5" drive cage cover and all 2.5" SATA drives](#addingremoving-25-storage-drives).
+    - The [CPU duct](#removing-the-cpu-duct) can optionally be removed to provide easier access to the cabling.
+2. Use the cutouts on the front right corner of the chassis to unplug the 4-pin Berg power connector (highlighted cyan below) and the two SATA data cables (highlighted green below) from the SATA backplane.
+
+![2.5" drive cage cabling & screws](./img/sata-backplane-cabling.webp)
+
+3. Unscrew the four screws holding the drive cage onto the chassis (highlighted red above), then remove the drive cage from the chassis.
+4. Unscrew the four SATA backplane screws from the drive cage, then push the SATA backplane out through the front of the drive cage.
+
+![SATA backplane screws](./img/sata-backplane-screws.webp)
+
+5. Place the new SATA backplane into the drive cage and secure it with the four SATA backplane screws, then reinstall the drive cage (if necessary).
+6. Plug the 4-pin Berg power connector and SATA data cables into the SATA backplane.
+    - The order of the SATA data cables shouldn't matter as long as your operating system and software is configured to address disks by UUID (e.g. `/dev/disk/by-id/...`) instead of by letter (e.g. `/dev/sd_`).
+    - By default, the `DATA1` port connects to SATA port 4 on the motherboard, and the `DATA0` port connects to SATA port 5.
+    - Ensure the cables are not plugged into SATA ports 2 and 3 (the bottom two ports), as these ports are unavailbable when the M.2 slots are fully populated.
```

Comment:
From the board it looks like the bottom M.2 is what turns those ports off **but** if you use an M.2 SATA and not an NVMe (NVMe for the other 3) those SATA ports will still work though. 

![screenshot-2024-07-08-18-44-29](https://github.com/system76/tech-docs/assets/4884946/9122791f-0b30-4a4f-abaa-649d13637b6c)

Reply:
It's irrelevant as long as someone doesn't plug into the bottom two ports, which never needs to happen because we only have two 2.5" slots. I'm not sure it's worth complicating the wording just to explain a situation that physically can't/doesn't need to happen in the chassis.

Reply:
Updated to match line 281 even though explaining this is completely pointless.

Reply:
Thank you and I do think it has a point if someone wants to add a PCIe device that uses SATA for power and will find those ports disabled but if they use an M.2 SATA instead of M.2 NVMe they will remain active.